adjust-bttn-size.js

```diff
@@ -1,19 +1,19 @@
 function changeButtonSize() {
-    // Get the selected size from the dropdown
-    const size = document.getElementById("button-size").value;
-    
-    // Get all buttons on the page
-    const buttons = document.querySelectorAll("button");
+  // Get the selected size from the dropdown
+  const size = document.getElementById("button-size").value;
```

Comment:
Use the `element` and `data` parameters to get the element and preference. Doing document.querySelectorAll will slow down because it is targeting elements multiple times, and getting the size this way simply won't work